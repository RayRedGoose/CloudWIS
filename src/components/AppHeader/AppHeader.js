import React from 'react'
import './AppHeader.scss'

const AppHeader = () => {
  return (
    <header className="app-header">
      <h1>
        <a href="https://github.com/RayRedGoose/CloudWIS">CloudWIS</a>
      </h1>
    </header>
  )
}

export default AppHeader
