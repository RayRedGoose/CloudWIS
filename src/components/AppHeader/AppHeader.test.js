import React from 'react'
import { shallow } from 'enzyme'
import AppHeader from './AppHeader'

it('should match the snapshot with all data passed in correctly', () => {
  const header = shallow(<AppHeader />)
  expect(header).toMatchSnapshot()
})
