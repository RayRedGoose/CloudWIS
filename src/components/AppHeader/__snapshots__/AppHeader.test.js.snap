// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot with all data passed in correctly 1`] = `
<header
  className="app-header"
>
  <h1>
    <a
      href="https://github.com/RayRedGoose"
    >
      CloudWIS
    </a>
  </h1>
</header>
`;
